provider/cluster_rosa_classic_resource.go

```diff
@@ -219,6 +219,12 @@ func (t *ClusterRosaClassicResourceType) GetSchema(ctx context.Context) (result
 				Type:        types.StringType,
 				Computed:    true,
 			},
+			"base_dns_domain": {
+				Description: "",
```

Comment:
Please fill in the description

Reply:
Fixed. Added docs as well...  it was missing for it.